.circleci/config.yml

```diff
@@ -56,6 +56,20 @@ jobs:
             name: postgres integration tests
             command: './scripts/circle-test-postgres.sh'
 
+  cache-server-test:
```

Comment:
I'm not sure about this. Slowing down the build, even more, feels wrong but at the same time, these things should be tested.

Reply:
Should definitely be tested.

Reply:
Perhaps we should add those contains to the backend test task rather than one just for cache servers. What do you think?

Reply:
Not sure tbh, maybe test it and see how it affects the time it takes to run backend-test?

Reply:
I'll keep it as it is now and I can experiment more with this later.